.github/workflows/test.yml

```
@@ -1,6 +1,7 @@
 name: test
-on: push
-
+on:
+  - push
+  - pull_request
```

Pull Requestが作成されたタイミングで `on.push` も発火してしまうので、`on.push` は デフォルトブランチのみに制限するのはどうでしょう？

```suggestion
  push:
    branches:
      - master 
  pull_request:
```

---

ありがとうございます〜！
おっしゃる通り、push と pull_request.open の2回でCIをフックしてしまいそうなので、[bc8c427](https://github.com/pepabo/active_merchant-epsilon/pull/124/commits/bc8c427189bbeaa7bd1062bdeee057570f3e8450) にて修正しました 🙌